src/seedu/addressbook/data/person/Phone.java

```diff
@@ -56,4 +56,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
+        return "Phone: " + value;
```

Comment:
Just a small thing. Perhaps, you can consider using `String.format()` to format `"Phone: " + value`